html/modules/custom/reliefweb_job_tagger/reliefweb_job_tagger.module

```diff
@@ -400,8 +401,24 @@ function reliefweb_job_tagger_queue_job(Node $job) : void {
 /**
  * Index embedding for a job.
  */
-function reliefweb_job_tagger_index_embedding($nid) {
-  /** @var \Drupal\ocha_ai_tag\Services\OchaAiTagTagger $jobTagger */
+function reliefweb_job_tagger_index_embedding(ContentEntityInterface $node) {
+  $entity_type_id = $node->getEntityTypeId();
+  if ($entity_type_id !== 'node') {
+    return;
+  }
+
+  $data = [
+    'id' => $node->id(),
+    'title' => $node->get('title')->value,
+    'url' => $node->toUrl('canonical')->setAbsolute()->toString(),
+    'contents' => [
+      'body' => [
+        'type' => 'markdown',
+        'content' => $node->get('body')->value,
```

Comment:
The title is not prepended here, which is different than what the RW source plugin does: https://github.com/UN-OCHA/ocha_ai/blob/a08ff287b3d326b8baa7914850f8efad295ee293/src/Plugin/ocha_ai/Source/ReliefWeb.php#L1088.